### DISCUSSION TITLE
Rendering SF Symbols?

### DISCUSSION THREAD

**Discussion by sbaildon** [kovidgoyal/kitty#6093] [Q&A] [answered]
macOS has access to [SF Symbols](https://developer.apple.com/sf-symbols/), which Apple's `Terminal.app` can render; is this something kitty can support?

<img width="262" alt="image" src="https://user-images.githubusercontent.com/214264/223107209-38ff15a9-e363-4fb1-ace8-6153b94f7769.png">

**Comment by page-down** [comment, marked as answer]
Nothing special, install fonts from SF Symbols.
If the font cannot be applied automatically, you can use `symbol_map` to map it manually.

```shell
kitty --debug-font-fallback -o font_size=20 -o 'symbol_map U+F8FF,U+100000-U+1018C7 SF Pro'

# print the apple symbol
python -c 'print(u"\uf8ff \U001008fa ")'
```

https://sw.kovidgoyal.net/kitty/faq/#kitty-is-not-able-to-use-my-favorite-font

Note that these are not mono fonts.
Open `Font Book`, `Fixed Width` and you can see that the relevant SF Symbols fonts are not listed in it, so those fonts are not mono fonts. Those will not be listed in `kitty +list-fonts`.
